ui/app/registration/views/patientcommon.html

```diff
@@ -229,7 +241,7 @@
             <div class="identifier-ui"  ng-hide="!showExtIframe">
                 <div class="identifier-iframe">
                     <span class="close" ng-click="closeIdentifierPopup()">&times;</span>
-                    <iframe name="identifier-verification" id="identifier-popup" height="95%" width="100%"></iframe>
+                    <iframe id="ndhm-popup" height="95%" width="100%"></iframe>
```

Comment:
The name NDHM/ABDM should not be hard coded anywhere in Bahmni product codebase. Can this pop-up be called: "identifer-extension-popup"?

Reply:
done. its renamed